### ISSUE TITLE
Crash report from Dynamo 2.10.1.3976

### ISSUE THREAD

**Issue by Nils-kun** [DynamoDS/Dynamo#11853]
# Issue Description
Please fill in the following information to help us reproduce the issue:

### What did you do?
Started new script in dynamo, after adding 2 nodes Dynamo crashes. Repeatability 100%.

### What did you expect to see?
Dynamo not crashing.

### What did you see instead?
Dynamo crashing

### What packages or external references (if any) were used?
Several are installed but none were used, Dynamo crashed with basic nodes.

### Stack Trace
```
Set connectionId threw an exception.

   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at Dynamo.UI.Controls.PreviewControl..ctor(NodeViewModel nodeViewModel)
   at Dynamo.Controls.NodeView.CreatePreview(NodeViewModel vm)
   at Dynamo.Controls.NodeView.TryShowPreviewBubbles()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

---
OS: `Microsoft Windows NT 10.0.19043.0`
CLR: `4.0.30319.42000`
Dynamo: `2.10.1.3976`

**Comment by QilongTang** [comment]
Another bug worth tracking and fixing

**Comment by QilongTang** [comment]
Tracked as https://jira.autodesk.com/browse/DYN-3945 and scheduled to fix as part of Dynamo 2.13.0
